File: frontend/src/app/dtos/event/event-details-shows.dto.ts
export class EventDetailsShowsDto {
  constructor(
    public id: number,
    public eventId: number,
    public eventTitle: string,
    public date: string,
    public time: string,
    public location: string,
    public room: string,
    public lowestPrice: number,
    public highestPrice: number,
    public bookedOut: boolean
  ) {}
}

File: backend/src/main/java/at/ac/tuwien/sepm/groupphase/backend/endpoint/dto/event/EventDetailsShowsDto.java
package at.ac.tuwien.sepm.groupphase.backend.endpoint.dto.event;

import java.math.BigDecimal;

public record EventDetailsShowsDto(
    Long id,
    Long eventId,
    String eventTitle,
    String date,
    String time,
    String location,
    String room,
    BigDecimal lowestPrice,
    BigDecimal highestPrice,
    boolean bookedOut) {}
